alerts/cluster-autoscaler-operator/OWNERS

```diff
@@ -0,0 +1,9 @@
+component: "Cloud Compute"
+subcomponent: "Other Providers"
+approvers:
+  - enxebre
+  - frobware
+  - michaelgugino
```

Comment:
we can remove `frobware` and `michaelgugino`, and add `lobziik` and `fedosin` please

Reply:
@elmiko I copied this file from https://github.com/openshift/cluster-autoscaler-operator/blob/master/OWNERS.. should we possibly update things there?

Reply:
yeah definitely, i can propose a patch there

Reply:
@cblecker just fyi, this is the updated list https://github.com/openshift/cluster-autoscaler-operator/pull/231

Reply:
perfect! I'll update it here too